docs/guides/elements-lookup-troubleshooting.md

```diff
@@ -146,3 +146,13 @@ The final recommendation there would be:
   to improve the corresponding WebDriverAgent sources, but keep in mind there are many automation
   tests around that rely on the current way these attributes are calculated, and we probably don't
   want to break them.
+
+## Symptom #4
+
+The desired element is shown in the page tree, but its property value contains partial value. e.g. if the element's `value` attribute had 600 bytes words, the XML page tree includes only 512 bytes. Rest of them are cut off.
+
+## Resolutions To Symptom #4
+
+Get the element attribute after getting the element. For example, getting the element with `findElement` and getting the `value` from the element attribute. The individual attribute gets the element attribute by getting snapshot deeply to return the entire value. Please see [this issue](https://github.com/appium/appium-xcuitest-driver/issues/2552) and [this PR](https://github.com/appium/WebDriverAgent/pull/1007) about the details.
```

Comment:
Get the element attribute after getting the element. For example, getting the element with `findElement` and getting the `value` from the element attribute. The individual attribute gets the element attribute by getting snapshot deeply to return the entire value. -> Retrieve the element's value/text using the corresponding [API](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Reference/Commands/GetElementAttribute) to get the full/uncut content

Reply:
about the details -> for more details